config/sockets.js

```diff
@@ -25,7 +25,7 @@ module.exports.sockets = {
    * > https://sailsjs.com/docs/reference/web-sockets/socket-client           *
    *                                                                          *
    ***************************************************************************/
-  // transports: [ 'websocket' ],
+  transports: ['websocket'],
```

Comment:
I don't know if we explicitly needed this but we should leave this on to make sure we always are using websockets specifically.

I'd really like some more context on this setting but it does make things more predictable.